src/cell_environment.cpp

```diff
@@ -114,6 +114,10 @@ void CellEnvironment::_on_cell_death(Cell *cell) {
 	CellSpawner *spawner = Object::cast_to<CellSpawner>(this->get_parent());
```

Comment:
Line 113 `_lineageGraph.storeCell(cell);` is giving a warning while compiling: 
```
src\cell_environment.cpp(113): warning C4834: discarding return value of function with 'nodiscard' attribute.
```

I believe it's complaining about the fact that the storeCell function returns a Vector2 but then it isn't used anywhere here.